### PR TITLE
Feat/remove mocks

### PR DESCRIPTION
**Pull request by eliawk**
@bent0b0x could please have a look in the search test?

### CODE REVIEW

**Comment by bent0b0x** [comment]
@eliawk can we do this in separate shots/PRs? I think we need a more global solution to the search tests (i.e. creating and tearing down a mock DB) during the testing process. Not sure how simple it will be or if it belongs in this PR

**Comment by jellegerbrandy** [comment]
@bent0b0x , if you have some time, can you make the search tests work in this PR, so we can proceed with testing against the db again? Thanks :-)

File: package.json [hunk 1]
```diff
@@ -13,12 +13,13 @@
     "build:test": "yarn run clean && yarn run prep && NODE_ENV=test node --max_old_space_size=4096 node_modules/.bin/webpack --progress --colors",
     "build:staging": "yarn run clean && yarn run prep && yarn run compile:sass && NODE_ENV=staging node --max_old_space_size=4096 node_modules/.bin/webpack --progress --colors",
     "clean": "rm -rf build/",
-    "chimp-test": "yarn run chimp:setup & chimp chimp.config.js && yarn run chimp:teardown",
-    "chimp-circleci-test": "yarn run chimp:setup & chimp .circleci/chimp.circleci.config.js && yarn run chimp:teardown",
-    "chimp-test:watch": "yarn run chimp:setup & chimp chimp.config.js --watch && yarn run chimp:teardown",
```
Comment (bent0b0x) [inline package.json]: Can we delete these seemingly now unused scripts? `chimp:setup` and `chimp:teardown`

**Comment by bent0b0x** [comment]
@jellegerbrandy I'll take a look in the next day or so!

File: package.json (hunk 1 above)
Comment (bent0b0x) [reply package.json]: Done

**Comment by bent0b0x** [comment]
@jellegerbrandy I got everything working locally w/ the new system. However, I'm unable to get this to work on circle and I'm not sure why yet. I tried ssh-ing into the container and looks like I can't hit port `3000`; I don't think `mongoDB` is running but I'm not sure how to get it to do so.

I will look into it more a bit later but please let me know if you have any ideas immediately

**Comment by bent0b0x** [comment]
Branch is ✅ now

**Comment by jellegerbrandy** [comment]
Yes!

**Pull request merged**